Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset

```diff
@@ -33,8 +33,8 @@ MonoBehaviour:
   contentScenes:
   - Name: MRTKExamplesHubMainMenu
     Path: Assets/MRTK/Examples/Experimental/ExamplesHub/Scenes/MRTKExamplesHubMainMenu.unity
-    Included: 0
-    BuildIndex: -1
+    Included: 1
```

Comment:
I don't think we want to include this. It keeps getting overwritten by the default state of the repo